xml/security_ssh.xml

```diff
@@ -550,40 +527,150 @@ cd path                            Change remote directory to 'path'
+      </para>
     </step>
+    <step>
+      <para>
+      Create an RSA key pair with a custom name:
+    </para>
+    <screen>&prompt.user;<command>ssh-keygen -f <replaceable>backup-server-key</replaceable></command>
+      </screen>
+    </step>      
     <step>
      <para>
-      Enter a passphrase consisting of 10 to 30 characters. The same rules
-      as for creating safe passwords apply. It is strongly advised to
-      refrain from specifying no passphrase.
+       Create a Ed25519 key pair:
+     </para>
+     <screen>&prompt.user;<command>ssh-keygen -t ed25519</command>
+     </screen>
+     <para>
+       Ed25519 keys are fixed at 256 bits.
      </para>
     </step>
    </procedure>
-   <para>
-    You should make absolutely sure that the private key is not accessible
-    by anyone other than yourself (always set its permissions to
-    <literal>0600</literal>). The private key must never fall into the hands
-    of another person.
-   </para>
-   <para>
-    To change the password of an existing key pair, use the command
-    <command>ssh-keygen -p</command>.
-   </para>
+   
+    <para>
+      Host keys are managed a little differently. Private host keys must not 
+      have a passphrase, and the key pairs are stored in <filename>/etc/ssh</filename>.
+      The OpenSSH installer automatically generates a set of host keys, like 
+      the following example:
+    </para>
+    <screen>&prompt.user;<command>ls -l /etc/ssh</command>
+total 608
+-rw------- 1 root root 577834 2021-05-06 04:48 moduli
+-rw-r--r-- 1 root root   2403 2021-05-06 04:48 ssh_config
+-rw-r----- 1 root root   3420 2021-05-06 04:48 sshd_config
+-rw------- 1 root root   1381 2022-02-10 06:55 ssh_host_dsa_key
+-rw-r--r-- 1 root root    604 2022-02-10 06:55 ssh_host_dsa_key.pub
+-rw------- 1 root root    505 2022-02-10 06:55 ssh_host_ecdsa_key
+-rw-r--r-- 1 root root    176 2022-02-10 06:55 ssh_host_ecdsa_key.pub
+-rw------- 1 root root    411 2022-02-10 06:55 ssh_host_ed25519_key
+-rw-r--r-- 1 root root     96 2022-02-10 06:55 ssh_host_ed25519_key.pub
+-rw------- 1 root root   2602 2022-02-10 06:55 ssh_host_rsa_key
+-rw-r--r-- 1 root root    568 2022-02-10 06:55 ssh_host_rsa_key.pub</screen>
+
+    <para>
+      <command>ssh-keygen</command> has a special option, <option>-A</option>,
+      for creating new host keys. This creates new keys for each of the key 
+      types for which host keys do not exist, with the default key file path, 
+      an empty passphrase, default bit size for the key type, and default 
+      comment. This is useful when you want to replace all host keys at once,
+      or any selected key pairs. The following example creates a complete new 
+      set of keys:
+    </para>
+    <screen>&prompt.sudo;<command>rm /etc/ssh/ssh_host*</command>
+&prompt.sudo;<command>ssh-keygen -A</command></screen>
+    <para>
+      Then delete the new DSA keys (because DSA is deprecated). To replace 
```

Comment:
Rather than advising customers to delete the DSA keys that are generated, could we report an issue with openssh inside of suse that it shouldn't emit dsa keys unless requested?

Reply:
Sounds good, it's better than telling customers to delete them. Would you do it?

Reply:
opened https://github.com/openssh/openssh-portable/pull/303 for this. We should still keep this part in the docs for now

Reply:
I just noticed the default sshd_config omits the DSA key:

```
#HostKey /etc/ssh/ssh_host_rsa_key
#HostKey /etc/ssh/ssh_host_ecdsa_key
#HostKey /etc/ssh/ssh_host_ed25519_key
```

I will write it up accordingly.